### ISSUE TITLE
Improve dashboard table styling

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#8335]
There are probably more things that could be done to improve the way tables look in dashboards, but two simple things would be:

1. Have columns be flush left with each other (with some padding) rather than spaced out to take up all of the available space. The current behavior results in some dumb looking tables, like this:

![screen shot 2018-08-15 at 10 08 11 am](https://user-images.githubusercontent.com/2223916/44161941-95d13880-a073-11e8-9727-ef08fbe2bd7f.png)

1A: The fact that the question title isn't left-aligned with the first column heading is annoying.

2. Tighten up the height of rows to match that of query builder tables (QB rows are 30px tall, dashboard rows are closer to 38px). Query builder table as an example:

![screen shot 2018-08-15 at 10 08 35 am](https://user-images.githubusercontent.com/2223916/44162061-fbbdc000-a073-11e8-96e5-6086427cdbfe.png)

**Comment by lwqcz** [comment]
Yeah, some fixed or relative width settings of table columns would be great help to improve Dashboard look. And also a feature to change font family and text size. Just a few basic formating features 💛

**Comment by mazameli** [comment]
This would be a good follow-up for "Table Vibes" in 0.32, especially item 1.

**Comment by gsouf** [comment]
Thumb up. Finding myself struggled with unability to reproduce the assigned width of columns in the dashboard. Leading to very long columns for large string, forcing to scroll and making the table unusable.

**Comment by bcbob** [comment]
Yes being able to set the column width for questions in dashboard cards will be a HUGE plus.

**Comment by Antozzi** [comment]
Also a little bit more formatting options in table widget would be a big help!

**Comment by mazameli** [comment]
> Also a little bit more formatting options in table widget would be a big help!

Could you be more specific please, @Antozzi? Thanks!

**Comment by Antozzi** [comment]
Well, tables look very ugly, with a lot of whitespace around, especially
when you use only a couple of columns. Surely we need font size for columns
and rows, and probably need a totals row at the end of the table.

ср, 10 мар. 2021 г., 21:20 Maz Ameli <notifications@github.com>:

**Comment by mazameli** [comment]
@Antozzi, thanks. Re: a totals row, you may be interested in [the pivot table visualization we added in 0.38](https://www.metabase.com/blog/Metabase-0.38/index.html) a few weeks ago.
